teal/notifier/status.tl

```diff
@@ -54,6 +54,9 @@ function StatusModule._create_win()
       col = vim.o.columns,
       zindex = 10,
     })
+    local winhlns = api.nvim_create_namespace("notifier_winhl")
```

Comment:
Why creating a new name space? We aldreay have one, I think we should use it.

Reply:
Will do